src/extension-runners/base.js

```diff
@@ -0,0 +1,23 @@
+/* @flow */
+
+import type {ExtensionManifest} from '../util/manifest';
+
+export type Extension = {|
+  sourceDir: string,
+  manifestData: ExtensionManifest,
+|};
+
+export type ExtensionRunnerParams = {
+  extensions: Array<Extension>,
+  profilePath?: string,
+  keepProfileChanges: boolean,
+  startUrl: ?string | ?Array<string>,
+};
+
+export interface IExtensionRunner {
+  run(): Promise<void>,
+  reloadAllExtensions(): Promise<void>,
+  reloadExtensionBySourceDir(extensionSourceDir: string): Promise<void>,
+  registerCleanup(fn: Function): void,
+  exit(): Promise<void>
```

Comment:
yeah, definitely, and I always add the trailing comma, but I forgot to add it here, it is a pity that eslint doesn't check this on the flow types definitions.